scripts/examples/index.openlayers.attribution.wmts.3857.html

```diff
@@ -0,0 +1,238 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>OpenLayers WMTS Basemaps Demo</title>
+    <link
+        rel="stylesheet"
+        href="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.3.1/css/ol.css"
+        type="text/css"
+    />
+    <script src="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.3.1/build/ol.js"></script>
+  </head>
+
+  <body>
+    <div id="map" style="width: 800px; height: 600px;"></div>
+    <script>
+     // Copyied from @linzjs/geojson
+     /**
+      * Do two GeoJSON bounding boxes intersect?
+
+      * @param a must be in WGS84 coordinates
+      * @param b must be in WGS84 coordinates
+      */
+     function intersects(a, b) {
+         if (a[1] > b[3] || b[1] > a[3]) return false;
+
+         const a0 = a[0];
+         const b0 = b[0];
+         const a2 = a0 < a[2] ? a[2] : a[2] + 360;
+         const b2 = b0 < b[2] ? b[2] : b[2] + 360;
+
+         return (a0 <= b2 && b0 <= a2) || (a0 + 360 <= b2 && b0 <= a2 + 360) || (a0 <= b2 + 360 && b0 + 360 <= a2);
+     };
+
+     /**
+      * Normalize the value of a longitude to between -180 and 180 degrees.
+
+      * @param lon a longitude value than can be outside the range -180 to 180.
+      */
+     const normLon = (lon) => {
+         if (lon < -360) {
+             lon = -(-lon % 360);
+         } else if (lon > 360) {
+             lon = lon % 360;
+         }
+
+         if (lon < -180) {
+             return 360 + lon;
+         } else if (lon > 180) {
+             return lon - 360;
+         }
+
+         return lon;
+     };
+
+     function normExtent(extent) {
+         return [normLon(extent[0]), extent[1], normLon(extent[2]), extent[3]];
+     }
+     // end of @linzjs/geojson
+
+     class AttributionBounds {
```

Comment:
is this copy and pasted from somewhere?

Reply:
this is from `packages/linzjs-geojson/src/wgs84.ts`